### PR TITLE
fix: set generated: false if the connector doesnot return it

### PR DESCRIPTION
**Pull request by aaqilniz**
Fixes #[9897](https://github.com/loopbackio/loopback-next/issues/9897)

## Checklist

- [x] [Sign off your commits](https://loopback.io/doc/en/contrib/code-contrib.html) with DCO (Developer Certificate of Origin)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [x] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

**Comment by aaqilniz** [comment]
Hi, @dhmlau. Can you have a look at this quick PR?

**Review by dhmlau — APPROVED** [review]
@aaqilniz, thanks for the PR. LGTM. Would like to get one more approval before merging. Thanks.

**Pull request merged**
